File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
#include <algorithm>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int pcnt = 0;
    int ycnt = 0;
    pcnt += count(s.begin(), s.end(), 'p');
    pcnt += count(s.begin(), s.end(), 'P');
    ycnt += count(s.begin(), s.end(), 'y');
    ycnt += count(s.begin(), s.end(), 'Y');
    cout<<pcnt<<" "<<ycnt;
    if(pcnt == ycnt) answer = true;
    else answer = false;

    return answer;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    let p_cnt = 0, y_cnt = 0;
    let arr = s.split('');
    arr.map((x) => {
        if(x === 'p' || x === 'P') p_cnt++;
        if(x === 'y' || x === 'Y') y_cnt++;
    })

    return p_cnt === y_cnt ? true : false;
}